File: Finals/rev/triangle.py


def triangle(heigth,char):
    for i in range(0,heigth):
        for j in range(0,heigth-i-1):
            print(" ",end="")
        for k in range(0,2*i+1):
            print(char,end="")
        print()


def main():
    x=int(input("Enter rows : "))
    ch=input("Enter Character : ")
    triangle(x,ch)
    
main()

File: Finals/rev/triangle.cpp
#include<iostream>

using namespace std;

void triangle(int n,char ch)
{
    for(int i=0;i<n;i++)
    {
        for(int j=0;j<n-i-1;j++)
        {
            cout<<" ";
        }
        for(int k=0;k<2*i+1;k++)
        {
          cout<<ch;
        }

        cout<<endl;
    }
}

int main()
{
    int n;
    cout<<"Enter no of rows : ";
    cin>>n;
    char c;
    cout<<"Enter Character : ";
    cin>>c;
    triangle(n,c);
}